### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by RunGT** [fac20/week1-HKLO#27]
Great site guys!!

We have found an issue using tab for navigating the site. Tab will cycle through the nav bar lists but not the content. 

This has a knock on effect with ChromeVox. Meaning a partially sighted or blind user would have to use a mouse to click on content for the screen reader to voice the content.

**Comment by oliverjam** [comment]
I think this is a problem with the way we introduce Chromevox. Screenreaders have a different keyboard shortcut for navigating element by element through the page. Tab should only ever focus interactive elements like buttons or links (see [my issue here](https://github.com/fac20/Week-1-AJNP/issues/21) for more detail).

**Comment by RunGT** [comment]
> I think this is a problem with the way we introduce Chromevox. Screenreaders have a different keyboard shortcut for navigating element by element through the page. Tab should only ever focus interactive elements like buttons or links (see [my issue here](https://github.com/fac20/Week-1-AJNP/issues/21) for more detail).

Great thanks for this. I was just about to add tabindex to our HTML in a response to one of our issues for the same thing. Jack also talked it over with our group just now 😃

**Comment by oliverjam** [comment]
Yeah it's our fault for not introducing screenreaders more clearly. Hopefully my explanation on Slack makes sense
